exposee-app/Routes/users.js

```diff
@@ -0,0 +1,86 @@
+import express from 'express';
+import bcrypt from 'bcrypt';
+import { User } from '../models/users.js';
+import { Op } from 'sequelize';
+
+const router = express.Router();
+
+
+router.get ('/user', async(req,res)=> {
+  
+  
+  const users = await User.findAll();
+  console.log(users)
+  res.status(200).json(users);
+  
+     });
+  
+   
+// Route for user registration
+router.post('/user', async (req, res) => {
```

Comment:
your test plan only covers the get, you should have a test plan that covers this post as well! (you can either do video/gif or screenshots and add words)

for example
I hit the post route in postman (screenshot of result)
and then the database had a new record after as seen in the get route screenshot